examples/helpers-collection/main.go

```diff
@@ -91,13 +83,13 @@ func (c *Controller) ListCollections(ctx web.Context) error {
 		return errors.NewError(http.StatusBadRequest, err.Error())
 	}
 
-	return collection.Format(ctx, collection.Options{
+	return collection.FormatQuery(ctx, collection.Options{
 		Query:  c.DB.Model(&MyModel{}),
-		Filter: req,
-		Fetcher: func(query *orm.Query) (collection.Items, error) {
+		Filter: &req,
+		Fetcher: func(opts *collection.Options) (collection.Items, error) {
```

Comment:
explain?

Reply:
you decide what need, more variants

Reply:
explain, for what? like use case

Reply:
here, we need only query... with applied filters and sorts..